uobtheatre/productions/models.py

```diff
@@ -979,6 +982,14 @@ def total_tickets_sold(self) -> int:
             performance.total_tickets_sold() for performance in self.performances.all()
         )
 
+    @property
+    def excerpt(self) -> Optional[str]:
```

Comment:
excerpt feels like a slightly weird name for this? Is this not just a short description?

Reply:
https://www.google.com/search?q=excerpt - This is the common name given to a short-version of a description like this

Reply:
Yeah that definition is not what we are using it for. An excerpt is text extracted directly from the text. It's not a summary or description of it. Unless the idea is to let production teams add an excerpt from the script (in which case having the default value being the description makes no sense) this name is wrong.

Reply:
What do u want to call it then?

Reply:
description_short